scala/src/main/scala/smile/clustering/distance/Binary/MeanMahanttan.scala

```diff
@@ -0,0 +1,17 @@
+package smile.math.distance
+
+import scala.math.pow
+
+/**
+ * @author Beck Gaël
+ *
+ **/
+class MeanMahanttan extends BinaryDistance {
```

Comment:
What's the difference from regular Mahanttan distance?

Reply:
I don't know sorry

Reply:
[A survey of binary measurement.pdf](https://github.com/haifengl/smile/files/1777261/A.survey.of.binary.measurement.pdf)
Based on that survey difference between Manhattan and Mean Manhattan are as follow.
a,b,c,d are the resuts of the conteingency matrix
Manhattan : b + c
Mean Manhantan : (b + c) / (a + b + c + d)